openedx/frontend-app-course-authoring
# feat: update header to be keyboard accessible

JIRA Ticket: [TNL-10750](https://2u-internal.atlassian.net/browse/TNL-10750)

This PR resolves the menu overlap issue when changing between different menus in the header. Additionally this PR make the header keyboard accessible. The previous version of the header did not allow the Content, Settings, or Tools menus to be clickable/hoverable via the keyboard only. Lastly the PR makes the menus and button hover states consistent across all dropdown menus.

Testing

1. Check that all the links go to their expected page and none are missing
2. Check that all menus and links are accessible only using your keyboard.
